### PR TITLE
Update AD GPO Powershell Installer (WIN).ps1

### PR DESCRIPTION
**Pull request by HSSupport**
update to add:

optional message relay
example usage
remove the default entry for $InstallerLocation

### CODE REVIEW

**Comment by q2srw77** [comment]
Hi HS,

I can add in an argument Variable. Limiting it to Message Relay, which isn't commonly used doesn't seem to make sense. You can absolutely create a Branch from this base script with Message Relay which I assume is part of your use case requirements.

Thanks,

File: Windows/AD GPO Powershell Installer (WIN).ps1 [hunk 1]
```diff
@@ -44,12 +51,24 @@ else {
...
+	Exit 1
+	}
+else {
+	Write-Host "--InstallerLocation = "$InstallerLocation""
+	}
+
+# Check message host was filled in by end user
+if ($MessageRelayHost -eq "") {
+	Write-Host "No MessageRelay Set, continuing"}
+	}
```
Comment (q2srw77) [inline Windows/AD GPO Powershell Installer (WIN).ps1:66]: This Close Bracket needs to be removed on Line 66

Comment (HSSupport) [reply Windows/AD GPO Powershell Installer (WIN).ps1:66]: fixed, thanks for pointing out!

**Comment by HSSupport** [comment]
> Hi HS,
> 
> I can add in an argument Variable. Limiting it to Message Relay, which isn't commonly used doesn't seem to make sense. You can absolutely create a Branch from this base script with Message Relay which I assume is part of your use case requirements.
> 
> Thanks,

we have no requirement for limiting it to the message relay, so having the ability to add it in your variable sounds like a good way to give extra functionality.

**Comment by HSSupport** [comment]
goign to close this, delete the fork, and create a fork for minor improvements instead (so you can add the extra bit)

**Pull request closed without merging**